eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgPeersDb.scala

```diff
@@ -59,16 +59,22 @@ class PgPeersDb(implicit ds: DataSource, lock: PgLock) extends PeersDb with Logg
       statement.executeUpdate("CREATE INDEX removed_peer_at_idx ON local.peer_storage(removed_peer_at)")
     }
 
+    def migration45(statement: Statement): Unit = {
+      statement.executeUpdate("ALTER TABLE local.peers RENAME COLUMN data TO node_address")
+      statement.executeUpdate("ALTER TABLE local.peers ALTER COLUMN node_address DROP NOT NULL")
+      statement.executeUpdate("ALTER TABLE local.peers ADD COLUMN node_features BYTEA")
+    }
+
     using(pg.createStatement()) { statement =>
       getVersion(statement, DB_NAME) match {
         case None =>
           statement.executeUpdate("CREATE SCHEMA IF NOT EXISTS local")
-          statement.executeUpdate("CREATE TABLE local.peers (node_id TEXT NOT NULL PRIMARY KEY, data BYTEA NOT NULL)")
+          statement.executeUpdate("CREATE TABLE local.peers (node_id TEXT NOT NULL PRIMARY KEY, node_address BYTEA, node_features BYTEA)")
```

Comment:
I think the naming is confusing because there is a notion of "features from the `node_announcement`" in BOLT9, which are not what we are storing here. On top of that, `node_address` is indeed part of `node_annoucement` (and we do use the same codec), which adds to the confusion.

```suggestion
          statement.executeUpdate("CREATE TABLE local.peers (node_id TEXT NOT NULL PRIMARY KEY, node_address BYTEA, init_features BYTEA)")
```